CMake/RealmCore.cmake

```diff
@@ -167,3 +167,80 @@ function(build_existing_realm_core core_directory)
 
     define_built_realm_core_target(${core_directory})
 endfunction()
+
+function(build_realm_sync sync_directory)
```

Comment:
Does Android use cmake to build the object store? You might need to adjust your build configuration.